### ISSUE TITLE
tsconfig.paths not followed when validating path-aliases

### ISSUE THREAD

**Issue by FrederickEngelhardt** [YardenPorat/eslint-plugin-validate-declared-imports#1]
Hello thanks for adding this plugin! For react-native this plugin is very valuable for direct asset imports.

### Issue

tsconfig paths are not being considered valid path imports.

### Expected

tsconfig Paths aliases do not error when used to import svgs. 

### Example

Folder structure:
```
src
src/assets.d.ts
src/assets
src/assets/Spinner.svg
src/component/Component.tsx
```

tsconfig.json
```json
{
  "compilerOptions": {
    "baseUrl": "./",
    "paths": {
      "assets/*": ["src/assets/*"]
    }
  },
}
```

declarations
```typescript
declare module '*.svg' {
  const content: React.FC<React.SVGAttributes<SVGElement>>
  export default content
}
````

eslint config (with react-native references)
```
module.exports = {
  root: true,
  extends: [
    '@react-native-community', // adds typescript support in this config
    'prettier',
  ],
  plugins: ['validate-declared-imports'],
  rules: {
      "validate-declared-imports/no-unresolved-declared-imports": 
          ["error", { 
              "fileExtensions": [
                  ".svg", // svg
                  ] 
                }
          ]
        }
}
```

Component.tsx
```
// opens a definitions window and shows both the relative path import and declaration file
import ../assets/Spinner.svg

// Only resolve the assets.d.ts file
import assets/Spinner.svg

// bad imports that do not error (until runtime)
import ../assets/Spinner1.svg
import ../assets/123/Spinner.svg
```

**Comment by FrederickEngelhardt** [comment]
There is a PR opened to address this issue.

**Comment by FrederickEngelhardt** [comment]
Resolved in #4 🎉
